nixos/modules/services/networking/syncthing.nix

```diff
@@ -135,27 +135,27 @@ in {
 
             id = mkOption {
               type = types.str;
-              description = ''
-                The device ID. See <link xlink:href="https://docs.syncthing.net/dev/device-ids.html"/>.
+              description = mdDoc ''
+                The device ID. See [](https://docs.syncthing.net/dev/device-ids.html).
               '';
             };
 
             introducer = mkOption {
               type = types.bool;
               default = false;
-              description = ''
+              description = mdDoc ''
                 Whether the device should act as an introducer and be allowed
                 to add folders on this computer.
-                See <link xlink:href="https://docs.syncthing.net/users/introducer.html"/>.
+                See [](https://docs.syncthing.net/users/introducer.html).
```

Comment:
The [auto link-text extension](https://nixos.org/manual/nixpkgs/stable/#ssec-contributing-markup-automatic-links) only really works for in-document cross-references. The markdown syntax for external links is `<https://docs.syncthing.net/users/introducer.html>` (or just leaving them as is, when unambiguous).

Reply:
leaving them as is has not worked in this particular instance, but we'll try the `<>` variant and change it to that if successful. thank you